daemon/daemon/daemon.go

```diff
@@ -9,16 +9,27 @@ import (
 
 	"github.com/40acres/40swap/daemon/database"
 	"github.com/40acres/40swap/daemon/database/models"
+	"github.com/40acres/40swap/daemon/lightning"
 	"github.com/40acres/40swap/daemon/rpc"
 	"github.com/40acres/40swap/daemon/swaps"
 	log "github.com/sirupsen/logrus"
 )
 
 const MONITORING_INTERVAL_SECONDS = 10
 
-func Start(ctx context.Context, server *rpc.Server, db database.SwapInRepository, swaps swaps.ClientInterface, network rpc.Network) error {
+func Start(ctx context.Context, server *rpc.Server, db database.SwapInRepository, swaps swaps.ClientInterface, network lightning.Network) error {
 	log.Info("Starting 40swapd")
 
+	config, err := swaps.GetConfiguration(ctx)
+	if err != nil {
+		return err
+	}
+	if config.BitcoinNetwork != network {
+		return fmt.Errorf("network mismatch: expected %s, got %s", network, config.BitcoinNetwork)
+	}
+
+	log.Infof("Network is %s", network)
```

Comment:
NIT; Shouldnt the network be on the `Starting 40swapd` log? Like `Starting x on network y`?

Reply:
Sure, I can merge both logs

Reply:
fixed